src/qibocal/protocols/characterization/resonator_spectroscopy.py

```diff
@@ -141,47 +176,75 @@ def _acquisition(
             ResSpecType,
             (qubit),
             dict(
-                signal=result.magnitude,
-                phase=result.phase,
+                signal=np.abs(result.average.voltage),
+                phase=np.mean(result.phase, axis=0),
                 freq=delta_frequency_range + ro_pulses[qubit].frequency,
+                error_signal=result.average.std,
+                error_phase=np.std(result.phase, ddof=1),
```

Comment:
```suggestion
                error_phase=np.std(result.phase, axis=0, ddof=1),
```
Wouldn't it be better to compute error bars for each data point (averaging across `nshots`)? If so, the same applies to the other routines.

Reply:
You are right, Thanks!